app/models/user.rb

```diff
@@ -0,0 +1,7 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+  has_many :burrows, through: :user_burrows
```

Comment:
@keelanpattersoncaesar @Curtisaliu hey guys! check out the User model, it looks different than what I saw you working on. Maybe it didn't save right, or got lost during the copy/paste?

Reply:
Thanks Nick! Yea I don't think it saved properly, we will fix it soon.